ASP/HTML-ASP.sublime-syntax

```diff
@@ -14,9 +14,10 @@ contexts:
         - match: "%>"
           scope: punctuation.section.embedded.end.asp
           pop: true
-        - match: (').*?(?=%>)
-          scope: comment.line.apostrophe.asp
-          captures:
-            1: punctuation.definition.comment.asp
-        - include: scope:source.asp
+        - match: ''
+          push:
+            - include: scope:source.asp # don't directly push, so that the scope doesn't also contain 'source.asp ' as well as our 'source.asp.embedded.html '
+          with_prototype:
+            - match: "(?=%>)"
+              pop: true
```

Comment:
As you have expressed your interest (@jcberquist and @michaelblyons), you may want to know that I have [created a PR](https://github.com/sublimehq/Packages/pull/367) to make the changes for remembering the scope when moving between embedded HTML and ASP. So far it doesn't support being inside script or style tags etc. but its a step in the right direction, and I'd welcome your input on how it can be improved :)